docs/book/cookbook/application-wide-layout.md

```diff
@@ -0,0 +1,85 @@
+# Set Application-Wide Layout
```

Comment:
I think this should be "Set the Application-Wide Layout"

Reply:
The navigation entry and the first headline are always the same, therefore I used the _short_ version without "the" here.